manpages/lumina-open.8

```diff
@@ -0,0 +1,106 @@
+.Dd November 6, 2017
+.Dt LUMINA-OPEN 1
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-open
+.Nd a graphical front-end to xdg-open.
+It is used to open files or links with the proper application.
+
+.Sh SYNOPSIS
+.Nm
+.Pp
+.Nm
+.Op Fl select | action Ar ActionID
+.Op Ar FILE_PATH | URL
+.Pp
+.Nm
+.Cm [-volumeup | -volumedown | -brightnessup | -brightnessdown |
+.Cm -autostart-apps | -terminal]
```

Comment:
These .Cm entries should probably be reworked as .Fl entries, as they are technically flags, not commands.